src/Components/table/table.component.tsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import {
+  Panel,
+  PanelBody,
+  PanelFooter,
+  PanelHeader,
+  TeamsThemeContext,
+  Dropdown,
+  IconButton
+} from "msteams-ui-components-react";
+import { Table, TBody, Tr, Th, Td } from "msteams-ui-components-react";
+import {
+  MSTeamsIcon,
+  MSTeamsIconType,
+  MSTeamsIconWeight
+} from "msteams-ui-icons-react";
+import "./table.styles.scss";
+interface Props { }
+
+interface State {
+  data: {
```

Comment:
create interface for the data item 
for example : 
 interface ICallItem {
   avatar: string;
    name: string;
    type: string;
    duration: string;
    date: string;
    three: number;
}

then the state will be like : 
interface State {
   data: ICallItem [] ;
}